tree/ntuple/v7/src/RPageStorageFile.cxx

```diff
@@ -220,7 +220,77 @@ ROOT::Experimental::Detail::RPageSourceFile::RPageSourceFile(std::string_view nt
       *fMetrics.MakeCounter<RNTuplePlainCounter*> ("timeWallUnzip", "ns", "wall clock time spent decompressing"),
       *fMetrics.MakeCounter<RNTupleTickCounter<RNTupleAtomicCounter>*>("timeCpuRead", "ns", "CPU time spent reading"),
       *fMetrics.MakeCounter<RNTupleTickCounter<RNTuplePlainCounter>*> ("timeCpuUnzip", "ns",
-                                                                       "CPU time spent decompressing")
+                                                                       "CPU time spent decompressing"),
+      *fMetrics.MakeCounter<RNTupleCalcPerf*> ("bwRead", "MiBi/s", "bandwidth compressed bytes read per second",
+         fMetrics, [](const RNTupleMetrics &metrics) -> std::pair<bool, double> {
+            if (const auto szReadPayload = metrics.GetCounter("RPageSourceFile.szReadPayload")) {
+               if (const auto szReadOverhead = metrics.GetCounter("RPageSourceFile.szReadOverhead")) {
+                  if (const auto timeWallRead = metrics.GetCounter("RPageSourceFile.timeWallRead")) {
+                     if (auto walltime = timeWallRead->GetValueAsInt()) {
+                        double payload = szReadPayload->GetValueAsInt();
+                        double overhead = szReadOverhead->GetValueAsInt();
+                        // unit: bytes / nanosecond = GB/s; MiBi = GB * 1000 / 1.024 / 1.024
+                        return {true, 1000. * (payload + overhead) / (1.024 * 1.024 * walltime)};
```

Comment:
```suggestion
                        return {true, (payload + overhead) / (1.000 * walltime)};
```

Reply:
I *think* that's wrong: walltime is in ns, payload + overhead in byes, so we're talking b/ns = GB/s. To get MB/s I need to multiply with 1000, not divide by 1000?

Reply:
Ouch. Yes.